cmd/down.go

```diff
@@ -181,3 +194,14 @@ func stopSyncthing(dev *model.Dev) {
 		log.Infof("failed to delete existing syncthing folder")
 	}
 }
+
+func waitForDevPodTermination(c kubernetes.Interface, p *v1.Pod) {
```

Comment:
Could we wait for all pods with label `interactive.dev.okteto.com: dev.Name` or `detached.dev.okteto.com: dev.Name`?

Reply:
yes, let me look into this. there are a couple other issues as well.